src/mock-data/user.json

```diff
@@ -0,0 +1,18 @@
+{
+    "PassengerInformation":{
```

Comment:
all names in .json file should start from small letter: passengerInformation, age, discount and etc